rjc_account_invoice_reimbursable/tests/test_reimbursable.py

```diff
@@ -0,0 +1,135 @@
+from odoo.tests.common import TransactionCase
+from odoo import fields
+
+
+class TestReimbursable(TransactionCase):
```

Comment:
If you don't use this, please remove.
But it would be nice if you can add another test script for itself.